app/templates/components/preprint-form-project-select.hbs

```diff
@@ -92,7 +92,7 @@
                                             {{/if}}
                                         {{/if}}
                                         <form onchange={{action 'track' 'input' 'onchange' 'Preprints - Submit - Add title, Connect preprint to existing project'}}>
-                                            {{preprint-title-editor nodeTitle=nodeTitle titleValid=titleValid}}
+                                            {{preprint-title-editor nodeTitle=nodeTitle titleValid=titleValid pressSubmit=(action (if (eq convertOrCopy 'convert') existingNodeExistingFile createComponentCopyFile) name)}}
```

Comment:
I see... this copies logic from line 99. I'd say let this pass, but note the copy-pasted logic it as a potential maintainability issue for the future.